stacks/analytics-ingest-lambda.yml

```diff
@@ -319,3 +384,51 @@ Resources:
       Dimensions:
         - Name: FunctionName
           Value: !Ref AnalyticsRedisLambdaFunction
+  AnalyticsDynamodbErrorAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[AnalyticsLambda][DynamoDB][Errors] ${EnvironmentType} > 0"
```

Comment:
I've been wondering if `> 0` for lambda errors is too noisy, even for `WARN`, because of the automatic retrying. The first retry happens almost immediately, so `> 1` in 5 minutes would do a decent job of filtering out the transient (single request) issue. Most of the events that trigger these alarms have probably successfully completed before we even get the alarm.